plugins/jobs/src/js/JobsOverview.tsx

```diff
@@ -2,28 +2,24 @@ import * as React from "react";
 import Router, {
   InjectedRouter,
   LocationDescriptor,
-  Link,
   withRouter
 } from "react-router";
 
-// tslint:disable-next-line:no-submodule-imports
 import { BehaviorSubject } from "rxjs/BehaviorSubject";
-// tslint:disable-next-line:no-submodule-imports
+import { Observable, Subscribable } from "rxjs/Observable";
```

Comment:
This impor for rxjs is missing the operations that are used on this file. Like startWith for instance.

```
 import "rxjs/addoperator/startWith";
```

Reply:
fixed below